low-level/files/opt/seagate/sspl/conf/sspl.config.conf.tmpl

```diff
@@ -0,0 +1,45 @@
+cortx:
+  release:
+    product: LR_R2
+    setup: cortx
+server_node:
+  30512e5ae6df9f1ea02327bab45e499d:
```

Comment:
@ujjwalpl Variable key here i.e {machine-id} needs to be prepended with TMPL_ ?
Same case for such variable key? And Component CI would change it to actual config value (without TMPL_ ofcourse)!

Also value for every leaf key (lowest level key in hierarchy) needs to be updated, need to prepend TMPL_ there too? Dont see any value there, unless over simplifying for CI pipeline.

Reply:
Added TMPL_ wherever required. Please review.